osu.Game/Overlays/Wiki/WikiHeader.cs

```diff
@@ -5,14 +5,18 @@
 using System.Linq;
 using osu.Framework.Bindables;
 using osu.Framework.Graphics;
+using osu.Framework.Localisation;
+using osu.Game.Localisation;
 using osu.Game.Online.API.Requests.Responses;
 
 namespace osu.Game.Overlays.Wiki
 {
     public class WikiHeader : BreadcrumbControlOverlayHeader
     {
-        private const string index_page_string = "index";
         private const string index_path = "Main_Page";
+        public static LocalisableString IndexPageString => WikiStrings.IndexPageString;
+        public static LocalisableString HeaderTitle => WikiStrings.HeaderTitle;
+        public static LocalisableString HeaderDescription => WikiStrings.HeaderDescription;
```

Comment:
I don't really see why this is declared twice, when you could just prepend `WikiStrings` in every usage, making it read even better. (same applies to `NewsHeader`)

Reply:
Full PR Reference:

[1] [LocalisableString usage](https://github.com/peppy/osu/blob/master/osu.Game/Overlays/ChatOverlay.cs#L38-L39)
```csharp
        public LocalisableString Title => ChatStrings.HeaderTitle;
        public LocalisableString Description => ChatStrings.HeaderDescription;
```

[2] [Code format of `*Strings.cs`](https://github.com/ppy/osu/pull/12495#issue-619104156)
```csharp
/// <summary>
/// "Fallback / default text"
/// </summary>
public static LocalisableString LookupKey => new TranslatableString(getKey("lookup_key"), "Fallback / default text");
```

p.s. will remove `static` from `*Header.cs` to match [1]

Reply:
The difference there is that `ChatOverlay` was *forced* to redeclare those members, because of implementing `INamedOverlayComponent`.

That said this PR has several places where the same string is referred to multiple times, so I'm not overly fussed about this myself.

Reply:
I see.